extensions/common/xwalk_external_extension.h

```diff
@@ -53,7 +54,7 @@ class XWalkExternalExtension : public XWalkExtension {
 
   // Variables from the browser process. Usually things like currently-running
   // application ID.
-  base::ValueMap runtime_variables_;
+  base::DictionaryValue::Storage runtime_variables_;
```

Comment:
I'm still not sure about the use of DictionaryValue::Storage as I don't see it being used anywhere upstream. May I know why you chose DictionaryValue::Storage to replace ValueMap.